### ISSUE TITLE
Implement Glossification in CK Editor 5

### ISSUE THREAD

**Issue by santwuma** [NCIOCPL/cgov-digital-platform#4049]
We need to implement glossification in the WYSIWYG - including implementing how it will connect to the glossification service to glossify the terms.

Example Cancer.gov pages with glossified terms:
https://www.cancer.gov/about-cancer/understanding/what-is-cancer
https://www.cancer.gov/about-cancer/treatment

### Requirements
- Things that stay the same:
  - On click of glossifier button, any term or synonym that is returned from the glossifier API is outlined and has checkmark
  - All instances of a term and its synonyms are outlined and have a checkmark, but the first instance is yellow instead of white
  - Once an item is selected, the box goes from yellow to white
  - Closing and reopening the glossifier displays the same terms/synonyms as glossifiable
  - Closing and reopening the glossifier displays the same terms/synonyms selected as glossified
  - Each glossified term gets the popup link with the CDRID

**Comment by laurelthrash** [comment]
Complete. Put on ODE and requires testing.

**Comment by alinai** [comment]
@sarinapadilla On ODE the following button appears as "Save" and on the existing CMS it appears as "OK" . Is it expected?
![image](https://github.com/NCIOCPL/cgov-digital-platform/assets/28455973/6ec872d8-ac99-4218-91c3-9038c6879411)

![image](https://github.com/NCIOCPL/cgov-digital-platform/assets/28455973/d960f830-5bf4-454b-9b25-4d15ad46ed1a)

**Comment by alinai** [comment]
@sarinapadilla , @santwuma , @KateMashkinaNIH  the glossification icon is missing on cms-test in both Intro Text and Content WYSIWYG.
![image](https://github.com/NCIOCPL/cgov-digital-platform/assets/28455973/283ed3ae-0ecc-4a7a-9bfa-6ff81bf314ca)
![image](https://github.com/NCIOCPL/cgov-digital-platform/assets/28455973/0e8b3678-4a04-44cc-ba28-6df256b366fd)

![image](https://github.com/NCIOCPL/cgov-digital-platform/assets/28455973/61486451-bf90-47ac-b3c9-34ca05075962)
![image](https://github.com/NCIOCPL/cgov-digital-platform/assets/28455973/52e469f2-ea6a-41d8-a0ef-c32bda42fb4f)

Entering it in the issue tracker

**Comment by alinai** [comment]
> @sarinapadilla On ODE the following button appears as "Save" and on the existing CMS it appears as "OK" . Is it expected? ![image](https://user-images.githubusercontent.com/28455973/271750310-6ec872d8-ac99-4218-91c3-9038c6879411.png)
> 
> ![image](https://user-images.githubusercontent.com/28455973/271750196-d960f830-5bf4-454b-9b25-4d15ad46ed1a.png)

@sarinapadilla , please confirm the above.

**Comment by alinai** [comment]
@sarinapadilla , the href and data-glossary-id positions are interchanged under the <a> tag. Are you ok with it?

ode: 
<a class="definition" href="/Common/PopUps/popDefinition.aspx?id=CDR0000445043&amp;version=Patient&amp;language=en" data-glossary-id="CDR0000445043">lung cancer</a>

cms-test:
<a class="definition" href="/Common/PopUps/popDefinition.aspx?id=CDR0000445043&amp;version=Patient&amp;language=en" data-glossary-id="CDR0000445043">lung cancer</a>

cms-dev: 
<a class="definition" data-glossary-id="CDR0000445043" href="/Common/PopUps/popDefinition.aspx?id=CDR0000445043&amp;version=Patient&amp;language=en">lung cancer</a>

**Comment by sarinapadilla** [comment]
> @sarinapadilla On ODE the following button appears as "Save" and on the existing CMS it appears as "OK" . Is it expected? ![image](https://user-images.githubusercontent.com/28455973/271750310-6ec872d8-ac99-4218-91c3-9038c6879411.png)
> 
> ![image](https://user-images.githubusercontent.com/28455973/271750196-d960f830-5bf4-454b-9b25-4d15ad46ed1a.png)

@alinai yes this looks correct!

**Comment by sarinapadilla** [comment]
> @sarinapadilla , the href and data-glossary-id positions are interchanged under the `<a>` tag. Are you ok with it?
> 
> ![image](https://user-images.githubusercontent.com/28455973/272594387-71875c59-e4e6-475a-a4af-41b74a9d1c5c.png)

@alinai yes, this is fine. As long as both of those properties are there, and the pop-up works when viewing the page and clicking the term, then it is okay.

**Comment by alinai** [comment]
Passed on ODE and cms-test
